aptos-move/framework/aptos-stdlib/doc/capability.md

```diff
@@ -14,7 +14,7 @@ EXPERIMENTAL
 ## Overview
 
 
-A capability is a unforgeable token which testifies that a signer has authorized a certain operation.
+A capability is an unforgeable token which testifies that a signer has authorized a certain operation.
```

Comment:
This is generated from move code, so you will need to fix the move code directly in the framework, to generate this